packages/mqtt/README.md

```diff
@@ -0,0 +1,27 @@
+# MQTT Publisher & Broker
+
+## Usage
+
+## Architecture
+
+### Error handling
+
+## Development
+
+Install dependencies:
+
+```
+npm i
+```
+
+To test:
+
+```
+npm run test
+```
+
+Lint:
+
+```
+npm run lint
+```
```

Comment:
Probably will have to update this file to just point to the docs once we have the MQTT setup specified in the Paima docs